WordPress/Classes/ViewRelated/Jetpack/Branding/Coordinator/JetpackBrandingCoordinator.swift

```diff
@@ -0,0 +1,20 @@
+import UIKit
+
+/// A class containing convenience methods for the the Jetpack branding experience
+class JetpackBrandingCoordinator {
+
+    static func presentOverlay(from viewController: UIViewController, redirectAction: (() -> Void)? = nil) {
+
+        let action = redirectAction ?? {
+            // TODO: Add here the default action to redirect to the jp app
+        }
+
+        let jetpackOverlayViewController = JetpackOverlayViewController(viewFactory: makeJetpackOverlayView, redirectAction: action)
+        let bottomSheet = BottomSheetViewController(childViewController: jetpackOverlayViewController, customHeaderSpacing: 0)
+        bottomSheet.show(from: viewController)
+    }
+
+    static func makeJetpackOverlayView(redirectAction: (() -> Void)? = nil) -> UIView {
```

Comment:
I think we can omit this function. `redirectAction` will always be needed I assume as without it we would be showing an overlay with a button that doesn't function. So calling this with the argument will be equivalent to calling directly the constructor.

Reply:
We certainly can, and we likely won't use it without action, but we might change the action (or use different actions for different instances), and if we use a method that returns a `UIView`, then the vc would be totally agnostic of the view. Wdyt?